### ISSUE TITLE
Remove attribute quotes

### ISSUE THREAD

**Issue by maltsev** [posthtml/htmlnano#6]
Source:

``` html
<div class="foo" title="hello world"></div>
```

Minified:

``` html
<div class=foo title="hello world"></div>
```

**Comment by voischev** [comment]
PR to posthtml-render? Or fork?

**Comment by maltsev** [comment]
I don't know yet. I was thinking about a more simple way: just to replace quotes in `result.html`.

**Comment by voischev** [comment]
ok

**Comment by kevincox** [comment]
I've opened https://github.com/posthtml/posthtml-render/pull/30 which should make this simple to solve.

**Comment by maltsev** [comment]
Thanks, @kevincox! Now we should just wait until a new version of `posthtml-render` would be released.

**Comment by kevincox** [comment]
FWIW it probably just makes sense to add a passthrough for render options. Then people can set existing options now and the new option once it is released. 

See https://github.com/parcel-bundler/parcel/pull/4212 for example. 

Even if you want to set this option by default it will be safely ignored on old versions of posthtml-render. 

TL;DR no need to wait.

**Comment by maltsev** [comment]
> FWIW it probably just makes sense to add a passthrough for render options. Then people can set existing options now and the new option once it is released.

Right. Thanks for the advice! I've added that: [fdfa0246](https://github.com/posthtml/htmlnano/commit/fdfa02465f37e72958f482f3976bd36ce0afef49).

**Comment by maltsev** [comment]
I guess we couldn't fully implement remove attribute quote minification since we have no control over posthtml-render when htmlnano is run as a PostHTML plugin. In that case, `quoteAllAttributes` attribute should be set by the developer themselves:

```js
posthtml([require('htmlnano')(options)])
    .process(html, {
        quoteAllAttributes: false
    })
```

I'll keep this issue open, maybe we'll find a better solution.

**Comment by SukkaW** [comment]
@maltsev Here is what I found:

First of all, when is it about to transform the AST tree back to HTML string, `tree.options` was passed to `posthtml-render`:

```js
// https://github.com/posthtml/posthtml/blob/c2dc7e62ce3b0bd7ba6502feedd9b133d3e99959/lib/index.js#L317
return render(tree, tree.options)
```

So what will happen when `posthtml-render` meets the option?

```js
// https://github.com/posthtml/posthtml-render/blob/93647af6399f4ef6bc8684b1731d799463c748de/lib/index.js#L51-L54
var quoteAllAttributes = options.quoteAllAttributes
if (typeof quoteAllAttributes === 'undefined') {
  quoteAllAttributes = true
}
```

Guess what? `quoteAllAttributes` will **only** be `true` if it is not presented in `tree.options`.

We can easily modify `tree.options` within a module. I will bring up a new PR and we can close the issue once for all.
